### PR TITLE
IE fix: Use image/jpeg also for jpg

### PR DESCRIPTION
**Pull request by sihu**

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -22,18 +22,23 @@ def generate(r):
         yield chunk
 
 def is_image(out_path):
-    ext = out_path.split('.')[-1]
-    return ext in ['jpg', 'jpeg', 'png', 'gif']
+    return extension(out_path) in ['jpg', 'jpeg', 'png', 'gif']
+
+def extension(out_path):
+    return out_path.split('.')[-1];
```
Comment (macav) [inline server.py]: unnecessary semicolon

File: server.py [hunk 2]
```diff
@@ -22,18 +22,23 @@ def generate(r):
...
+
+def mime_type_ending(out_path):
+    if extension(out_path) in ['jpg', 'jpeg']:
+        return 'jpeg'
+    return ext;
```
Comment (macav) [inline server.py]: `ext` is not defined here and semicolon

File: server.py [hunk 3]
```diff
@@ -22,18 +22,23 @@ def generate(r):
...
 
 def is_image(out_path):
-    ext = out_path.split('.')[-1]
-    return ext in ['jpg', 'jpeg', 'png', 'gif']
+    return extension(out_path) in ['jpg', 'jpeg', 'png', 'gif']
+
+def extension(out_path):
+    return out_path.split('.')[-1];
+
+def mime_type_ending(out_path):
```
Comment (sihu) [inline server.py]: we could also correctly name it `image_mime_type_ending` or take the `image` also in that function

**Review by sihu — COMMENTED** [review]
one comment

**Review by macav — CHANGES_REQUESTED** [review]

**Review by macav — APPROVED** [review]

**Pull request merged**